lib/rubygems/commands/rdoc_command.rb

```diff
@@ -86,8 +86,9 @@ def execute
       begin
         doc.generate
       rescue Errno::ENOENT => e
-        e.message =~ / - /
-        alert_error "Unable to document #{spec.full_name}, #{$'} is missing, skipping"
+        match = / - /.match(e.message)
+        alert_error "Unable to document #{spec.full_name}, " \
+                    " #{match.post_match} is missing, skipping"
```

Comment:
This one seems a different cop too, let's remove it from this PR.

Reply:
Unfortunately RuboCop thinks this is also `Performance/StringInclude` offense. It suggests calling `e.message.includes?(" - ")` since it doesn’t know about the following `$'`.

Would you rather we use `rubocop:disable Performance/StringInclude` around the method, instead?

Reply:
Aha! No, I think your alternative is better than what we have. I don't like using globals like this.